rules/falco_rules.yaml

```diff
@@ -1148,6 +1148,10 @@
 - macro: user_known_write_etc_conditions
   condition: proc.name=confd
 
+# This is a placeholder for user to extend the whitelist for write below etc rule
+- macro: user_know_write_below_etc_activities
```

Comment:
can you make this user_known instead of user_know?

Reply:
Done Thanks!